tests/unit/util/util.py

```diff
@@ -61,4 +61,40 @@ def convert_csv_to_parquet(csv_file: str, parquet_partition_name: str):
                         root_path=parquet_partition_name,
                         partition_cols=['start_year'])
 
+    print("End ", datetime.now())
+
+
+def convert_csv_to_single_parquet_file(csv_file: str, parquet_dir: str):
```

Comment:
This is a lot of code duplication, could this be done by adding a parameter to existing method?